src/resolvers/FlyerResolver.ts

```diff
@@ -238,6 +243,32 @@ class FlyerResolver {
       startDate,
       title,
     );
+    let editedResponse = '';
+    if ((endDate != null && location != null) || (location != null && startDate != null)) {
```

Comment:
Instead of checking if the variable is not equal to `null`, you can just use ther variable. For example, `if (endDate != null)` is equivalent to `if (endDate)`.